content/en/watchdog/insights.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Watchdog Insights
+kind: documentation
+description: View anomalies and outliers in context with Watchdog Insights
```

Comment:
We tend to use "in context" internally because we understand it as encompassing "that match user query" but it is worth making sure external users/readers have the clear info:

How about something like "View anomalies and outliers that match your search query with Watchdog Insights"?